src/kiwixapp.cpp

```diff
@@ -527,6 +527,7 @@ void KiwixApp::handleItemsState(TabType tabType)
     KiwixApp::instance()->getAction(KiwixApp::ZoomOutAction)->setDisabled(libraryOrSettingsTab);
     KiwixApp::instance()->getAction(KiwixApp::ZoomResetAction)->setDisabled(libraryOrSettingsTab);
     KiwixApp::instance()->getAction(KiwixApp::RandomArticleAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::OpenHomePageAction)->setDisabled(libraryOrSettingsTab);
```

Comment:
This change won't compile - there is no variable `app` yet. Please fix it by moving the last (code cleanup) commit to the beginning of the PR.

Reply:
Fixed, sorry for this.